code/modules/hydroponics/grown/cannabis.dm

```diff
@@ -117,5 +117,5 @@
 	name = "omega cannabis leaf"
 	desc = "You feel dizzy looking at it. What the fuck?"
 	icon_state = "ocannabis"
-	volume = 420
+	volume = 42.0
```

Comment:
This is less than the default 50 volume and makes no sense for a T2 plant.

Reply:
I wanted to preserve the joke but ok